### PR TITLE
NNEF to NNIR - Fix

### PR DESCRIPTION
**Pull request by hansely**

### CODE REVIEW

File: model_compiler/python/nnef_to_nnir.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import nnef
-import numpy as np
```
Comment (kiritigowda) [inline model_compiler/python/nnef_to_nnir.py]: We don't need this?

**Comment by kiritigowda** [comment]
This pull request **introduces 2 alerts** and **fixes 3** when merging 6cb429fdc5f24eb0bc1720f0523403734a230a1a into af4ec494c04a5ec10bd87e18b6ea267f13d0d9f2 - [view on LGTM.com](https://lgtm.com/projects/g/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/rev/pr-cbbe5f6ab04a6ae463ce66834d1a2d4ef4124a1f)

**new alerts:**

* 2 for Unused local variable

**fixed alerts:**

* 2 for Variable defined multiple times
* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by kiritigowda** [comment]
@hansely Can I merge this?

**Comment by hansely** [comment]
Yes, they are already included in nnir. So no need to import twice. You can merge it

**Pull request merged**